concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-24	24	Donner	Donner	thunder	1150	THUNDER
+Steinthal-1875-1565-25	25	Es donnert.	Es donnert.	to thunder	2378	THUNDER (VERB)
+Steinthal-1875-1565-26	26	Blitz	Blitz	lightning	1654	LIGHTNING
+Steinthal-1875-1565-27	27	Es blitzt.	Es blitzt.	Lightning strikes.		
+Steinthal-1875-1565-28	28	Es kommt ein Gewitter.	Es kommt ein Gewitter.	A thunderstorm is coming.		
+Steinthal-1875-1565-29	29	Der Regen hört auf.	Der Regen hört auf.	The rain stops.		
+Steinthal-1875-1565-30	30	Regenbogen	Regenbogen	rainbow	1733	RAINBOW
+Steinthal-1875-1565-31	31	Thau	Thau	dew	1977	DEW
+Steinthal-1875-1565-32	32	Schnee	Schnee	snow	784	SNOW
+Steinthal-1875-1565-33	33	Hagel	Hagel	hail	609	HAIL
+Steinthal-1875-1565-34	34	Reif	Reif	hoar frost	2942	HOARFROST
+Steinthal-1875-1565-35	35	Eis schmilzt, thaut.	Eis schmilzt.	Ice melts.		
+Steinthal-1875-1565-36	36	Eis schmilzt, thaut.	Eis thaut.	Ice thaws.		
+Steinthal-1875-1565-37	37	Tag, bricht an, geht zu Ende	Tag	day	1225	DAY (NOT NIGHT)
+Steinthal-1875-1565-38	38	Tag, bricht an, geht zu Ende	Tag bricht an.	daybreak		
+Steinthal-1875-1565-39	39	Tag, bricht an, geht zu Ende	Tag geht zu Ende.	Day comes to an end.		
+Steinthal-1875-1565-40	40	Nacht, sie kommt herein	Nacht	night	1233	NIGHT
+Steinthal-1875-1565-41	41	Nacht, sie kommt herein	Nacht kommt herein.	Night falls.		
+Steinthal-1875-1565-42	42	Morgen	Morgen	morning	1339	MORNING
+Steinthal-1875-1565-43	43	Mittag	Mittag	noon	12	MIDDAY
+Steinthal-1875-1565-44	44	Abend	Abend	evening	1629	EVENING
+Steinthal-1875-1565-45	45	Morgenroth	Morgenroth	dawn	729	DAWN
+Steinthal-1875-1565-46	46	Abendröthe	Abendröthe	dusk	2297	DUSK
+Steinthal-1875-1565-47	47	Es ist finster.	Es ist finster.	It is gloomy.	1552	GLOOMY
+Steinthal-1875-1565-48	48	Finsterniss	Finsternis	gloom		
+Steinthal-1875-1565-49	49	dunkel, -heit	dunkel	dark	706	DARK
+Steinthal-1875-1565-50	50	dunkel, -heit	Dunkelheit	darkness	11	DARKNESS
+Steinthal-1875-1565-51	51	hell	hell	bright	679	BRIGHT
+Steinthal-1875-1565-52	52	Licht	Licht	light	630	LIGHT (RADIATION)
+Steinthal-1875-1565-53	53	Schatten	Schatten	shadow	1388	SHADE
+Steinthal-1875-1565-54	54	Jahr	Jahr	year	1226	YEAR
+Steinthal-1875-1565-55	55	Jahreszeit	Jahreszeit	season	691	SEASON
+Steinthal-1875-1565-56	56	Sommer (trockne Zeit)	Sommer	summer	1342	SUMMER
+Steinthal-1875-1565-57	57	Sommer (trockne Zeit)	trockne Zeit	dry season	452	DRY SEASON
+Steinthal-1875-1565-58	58	Winter (Regenzeit)	Winter	winter	1471	WINTER
+Steinthal-1875-1565-59	59	Winter (Regenzeit)	Regenzeit	wet season	453	RAINY SEASON
+Steinthal-1875-1565-60	60	- ist nun vorüber.	Sommer, Winter ist nun vorüber.	Summer, winter is over now.		
+Steinthal-1875-1565-61	61	Herbst (Erntezeit)	Herbst	autumn	1276	AUTUMN
+Steinthal-1875-1565-62	62	Herbst (Erntezeit)	Erntezeit	harvest season	3327	AUTUMN OR HARVEST SEASON
+Steinthal-1875-1565-63	63	Frühling (Saatzeit)	Frühling	spring	1537	SPRINGTIME
+Steinthal-1875-1565-64	64	Frühling (Saatzeit)	Saatzeit	sowing time		
```

Comment:
Yeah, I mainly did this because of Winter (Regenzeit), and then felt I had to be consistent and split up all the seasons. Just because Winter and Regenzeit are such different concepts, climate-wise, and depending on where in the world this list is used, only one of them might really be applicable. And a harvest season might well exist even though there is no climatic difference between summer and autumn in certain regions.
(This issue comes up in other places in the list - sometimes, the item in brackets seemed to specify the previous one; and sometimes, it was so different that I felt the need to split them.)
If the reviewers and moderator still feel like these should be kept as a whole, I'll change it.